src/test/scala/com/webtrends/harness/component/akkahttp/RouteGeneratorTest.scala

```diff
@@ -38,26 +40,13 @@ class RouteGeneratorTest extends WordSpec with ScalatestRouteTest with Predefine
   implicit val logger: Logger = Logger.getLogger(getClass.getName)
 
 
-  "RouteGenerator " should {
-
-    def actorRef = actorSystem.actorOf(SimpleCommandActor())
-    def requestHandler(req: AkkaHttpRequest): Future[AkkaHttpRequest] = Future.successful(req)
-    def responseHandler200(resp: AkkaHttpRequest): Route = complete(StatusCodes.OK, resp.toString)
+  import RouteGeneratorTest._
 
-    def rejectionHandler: PartialFunction[Throwable, Route] = {
-      case ex: NotAuthorized => complete(StatusCodes.Unauthorized, ex.message)
-      case ex: Forbidden => complete(StatusCodes.Forbidden, ex.message)
-      case t: Throwable => complete(StatusCodes.InternalServerError, t.getMessage)
-    }
+  def actorRef = actorSystem.actorOf(TestCommandActor.createCommandActor(simpleFunction))
+  def messageActorRef = actorSystem.actorOf(TestCommandActor.createCommandActor(messageFunction))
+  def exceptionActorRef = actorSystem.actorOf(TestCommandActor.createCommandActor(exceptionFunction))
```

Comment:
Why doesn't this use wookiee-core CommandFactory? createCommandActor is still using wookiee-core's CommandActor class so this doesn't save us any external dependencies. Should just use the built in factory instead of recreating it in TestCommandActor.